lib/hocon/impl/config_null.rb

```diff
@@ -16,12 +16,12 @@ def transform_to_string
     "null"
   end
 
-  def render(sb, indent, atRoot, options)
-    sb.append("null")
+  def render_value_to_sb(sb, indent, at_root, options)
+    sb << "null"
   end
 
   def newCopy(origin)
```

Comment:
this should be `new_copy` (was already wrong, not part of your PR).  fixed in my local branch.